src/QMCWaveFunctions/SPOSet.cpp

```diff
@@ -112,6 +112,17 @@ void SPOSet::evaluateThirdDeriv(const ParticleSet& P, int first, int last, GGGMa
   APP_ABORT("Need specialization of SPOSet::evaluateThirdDeriv(). \n");
 }
 
+void SPOSet::evaluate_notranspose_spin(const ParticleSet& P,
+                                       int first,
+                                       int last,
+                                       ValueMatrix_t& logdet,
+                                       GradMatrix_t& dlogdet,
+                                       ValueMatrix_t& d2logdet,
+                                       ValueMatrix_t& dspinlogdet)
+{
+  evaluate_notranspose(P, first, last, logdet, dlogdet, d2logdet);
```

Comment:
This was my workaround, since the only place this function in SPOSets is called is in called in MultiDiracDeterminant::evaluateForWalkerMove. 
The intention was to allow this to give the right behavior in each case (spinors vs. normal).
If you have normal SPOs, you call this function and since the dspinlogdet isn't needed it just calls the normal evaluate_notranspose.
If you do have spinors, then it appropriately calculates the spin gradient and stores it. 

Since dspinlogdet is already of the right size (called by MultiDiracDeterminant::resize) I didn't call resize here.

This can definitely be cleaned up after the spinor flag is added to particleset. Working on that now

Reply:
This no longer has this behavior, and instead aborts if an SPOSet calls it. Only SpinorSet has an implementation